selenium_tests/pin/__init__.py

```diff
@@ -0,0 +1 @@
+__author__ = 'sam576'
```

Comment:
Can remove these (it's a PyCharm default, can be changed via Preferences > IDE Settings > File and Code Templates).